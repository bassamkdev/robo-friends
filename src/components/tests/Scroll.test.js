import React from 'react';

import Scroll from '../Scroll';

import { shallow } from 'enzyme';

it('render correctly Scroll component', () => {
    const wrapper = shallow(<Scroll children={'childrenComponent'}/>);
    expect(wrapper).toMatchSnapshot();
})