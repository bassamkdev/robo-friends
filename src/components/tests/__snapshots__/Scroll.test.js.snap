// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Scroll component 1`] = `
<div
  style={
    Object {
      "border": "2px solid black",
      "height": "700px",
      "overflowY": "scroll",
    }
  }
>
  childrenComponent
</div>
`;
